addons/core/translations/resources/en-us.yaml

```diff
@@ -311,6 +312,23 @@ target:
       add:
         title: Add Brokered Credential Sources
         description: Select brokered credential sources to assign to this target.
+  injected-application-credential-source:
+    title: Injected Application Credential
+    title_plural: Injected Application Credentials
+    types:
+      vault: Vault
+      username_password: Username & Password
```

Comment:
Not blocking, but I notice this and the ssh string appears more than a few times in this file, should it maybe live in a shared spot?

Reply:
I like that idea!  For now, it makes sense to me to keep it within resources and not use the 
[root](https://github.com/hashicorp/boundary-ui/blob/main/addons/core/translations/en-us.yaml) as these are just credential sources specific. 
One thing to note is that some of the occurrences have a `help` associated with them. Perhaps, we can add a new file within resources to move some of the most commonly used resource translations there? Open to suggestions!

Reply:
Merging this pr for now. But added this topic to our tech time to hear others' take on this.